script/verification/DisputeGameUpgrade.s.sol

```diff
@@ -86,4 +94,53 @@ abstract contract DisputeGameUpgrade is VerificationBase, SuperchainRegistry {
 
         require(address(faultDisputeGame.weth()) != address(permissionedDisputeGame.weth()), "weth-200");
     }
+
+    function precheckDisputeGames() internal view {
+        _precheckDisputeGameImplementation(GameType.wrap(0), expFaultDisputeGame);
+        _precheckDisputeGameImplementation(GameType.wrap(1), expPermissionedDisputeGame);
+    }
+
+    // _precheckDisputeGameImplementation checks that the new game being set has the same
+    // configuration as the existing implementation.
+    function _precheckDisputeGameImplementation(GameType _targetGameType, address _newImpl) internal view {
+        console.log("pre-check new game implementation", _targetGameType.raw());
+
+        FaultDisputeGame currentGame = FaultDisputeGame(address(dgfProxy.gameImpls(GameType(_targetGameType))));
+        FaultDisputeGame newGame = FaultDisputeGame(_newImpl);
+
+        if (vm.envOr("DISPUTE_GAME_CHANGE_WETH", false)) {
```

Comment:
seems that this is not ready the value `true` or `false` from the `DISPUTE_GAME_CHANGE_WETH` into the `.env`.
 If this is the case we should comment it to make sure in the future there is no issue failling with it.

Reply:
`envOr` with the second argument being a `bool` is reading the env var value as a boolean, see [docs](https://book.getfoundry.sh/cheatcodes/env-or). Or what do you mean?

Reply:
Thanks for confirming, I was incorrect I have tried this and this is working as expected like in the doc. 
I will resolve this comment.